packages/caip/src/adapters/coingecko/utils.ts

```diff
@@ -18,6 +18,8 @@ import {
   dogeChainId,
   ethAssetId,
   ethChainId,
+  gnosisAssetId,
+  gnosisChainId,
```

Comment:
Gnosis is GNO, not the native asset of the Gnosis chain. Make the AssetId `xdaiAssetId`?

Reply:
I'd still keep it as `gnosisAssetId` for consistency with others. The fact that xdai is the `gnosisAssetId` doesn't change the variable meaning